pkg/openthread/Makefile.include

```diff
@@ -8,3 +8,6 @@ ifneq (,$(filter openthread_contrib,$(USEMODULE)))
   DIRS += $(OPENTHREAD_DIR)/contrib
   DIRS += $(OPENTHREAD_DIR)/contrib/netdev
 endif
+
+# There is a problem with ranlib and LLVM/clang in this package
```

Comment:
Maybe also add `problem with unused '-mcpu' arguments and ranlib`

Reply:
But those are only warnings.

Reply:
Which I think may be the root cause of the `ranlib` error.